paypal/express/views.py

```diff
@@ -84,7 +84,7 @@ def get_redirect_url(self, **kwargs):
             # Transaction successfully registered with PayPal.  Now freeze the
             # basket so it can't be edited while the customer is on the PayPal
             # site.
-            basket.freeze()
+            #basket.freeze() # how to unfreeze this?
```

Comment:
@sjolicoeur This opens up exploits where users can add to the cart for free during checkout